### PR TITLE
Adding the empty view events in marionette.collectionview documentation

### PR DESCRIPTION
**Pull request by SauloSilva**
#2390

### CODE REVIEW

**Comment by jasonLaster** [comment]
Thanks. Mind squashing your commits

**Comment by SauloSilva** [comment]
@jasonLaster done

File: docs/marionette.collectionview.md [hunk 1]
```diff
@@ -742,6 +746,113 @@ The `before:render:collection` event is triggered before the `collectionView`'s
...
+myCollectionView.on("render:empty", function(){
+  alert("the empty view has been rendered");
+});
+
+myCollectionView.render()
+```
+
+### "before:remove:empty" event
+
+This is triggered after the empty view instance has been removed, but before it has been destroyed.
```
Comment (ahumphreys87) [inline docs/marionette.collectionview.md]: I think its called before it is removed as well. https://github.com/marionettejs/backbone.marionette/blob/minor/src/collection-view.js#L279-L284

Comment (SauloSilva) [inline docs/marionette.collectionview.md]: @ahumphreys87 makes sense, I ended change it.

**Comment by jasonLaster** [comment]
Looks great. Thanks!

**Pull request merged**